packages/tools/viewer-alpha/src/viewer.ts

```diff
@@ -17,7 +17,8 @@ import type { Nullable } from "core/types";
 import "core/Animations/animatable";
 import "core/Materials/Textures/Loaders/envTextureLoader";
 import "core/Helpers/sceneHelpers";
-import "loaders/glTF/2.0";
+// eslint-disable-next-line import/no-internal-modules
+import "loaders/glTF/2.0/index";

```

Comment:
@RaananW this doesn't seem right, but `import "loaders/glTF/2.0";` was getting converted to `import "loaders/glTF/2.0.js";` which clearly is not correct and broke at runtime. Is there some other way I'm supposed to deal with this?

Reply:
this is the right way to deal with it we do that in different public projects exactly due to this issue.